examples/network/img/indonesia/blossom.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?><svg id="Слой_1" height="128" width="128" style="enable-background:new 0 0 128 128;" version="1.1" viewBox="0 0 128 128" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><style type="text/css">
```

Comment:
To be honest: I don't see why we need this new image to show the new features. Maybe you can explain your intention?

Reply:
@mojoaxel 1) People should see that Vis allows to use SVG.
2) the image shows the correct way to use SVG, because of Firefox SVG drawing bug we must set width/height attributes inside the root `<svg>` element. 
I found the Firefox bug only when loaded SVG image into the example. 

I also added a note to Image section of Nodes docs here https://github.com/r3code/vis-network/commit/752e4b2711282d63da83156705e810688eed7b36

Reply:
IHMO this should be a separate PR. Your mixing two unrelated things.

Reply:
@Thomaash so you would like to see it in a history of the project separate?

Reply:
Yes, exactly.

Reply:
@mojoaxel I truned back package.json and package-lock.json https://github.com/r3code/vis-network/commit/2833cf24300a34ff5f0016b6ab22cb305fd4dce3

Reply:
@Thomaash I removed blossom.svg https://github.com/r3code/vis-network/commit/9345cf9cd14577ddea466062875892149e2a6e0c and fixed the example. I'll make a separate PR for that.